### PR TITLE
Fix: step group documentation update and bug fix of firstExecuteTime mission

### PR DESCRIPTION
**Pull request by zq200618**
Signed-off-by: Qiang Zheng <zhengq20018@cmbchina.com>

### Description of your changes

1. step group documentation update
2. bug fix of firstExecuteTime mission in step group execute status

I have:

- [ ] Read and followed KubeVela's [contribution process](https://github.com/kubevela/kubevela/blob/master/contribute/create-pull-request.md).
- [ ] [Related Docs](https://github.com/kubevela/kubevela.io) updated properly. In a new feature or configuration option, an update to the documentation is necessary. 
- [ ] Run `make reviewable` to ensure this PR is ready for review.
- [ ] Added `backport release-x.y` labels to auto-backport this PR if necessary.

### CODE REVIEW

**Review by wonderflow — APPROVED** [review]

File: docs/examples/workflow/step-group/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Step Group
+
+## How to start
+
+Edit a yaml file as `example.yaml`, then execute it with `kubectl apply` command.
```
Comment (FogDong) [inline docs/examples/workflow/step-group/README.md:5]:
```suggestion
Edit a yaml file as `example.yaml`, then execute it with `vela up` command.
```

File: docs/examples/workflow/step-group/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+## Parameter Introduction
+
+`step-group` have `subSteps` parameter which is an array contain any step type and its valid parameter exclude `step-group` step type itself
```
Comment (FogDong) [inline docs/examples/workflow/step-group/README.md:9]:
```suggestion
`step-group` has a `subSteps` parameter which is an array containing any step type whose valid parameters do not include the `step-group` step type itself.
```

File: docs/examples/workflow/step-group/README.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+`step-group` don't have `properties` parameter now
```
Comment (FogDong) [inline docs/examples/workflow/step-group/README.md:11]:
```suggestion
Note that `step-group` doesn't support `properties` for now.
```

File: docs/examples/workflow/step-group/README.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+## Execute process
+
+When executing the `step-group` step, the subSteps in the step group are executed in dag mode. The step group will only complete when all subSteps have been executed to completion.
+subStep have the same execute behaviour as it is a normal step
```
Comment (FogDong) [inline docs/examples/workflow/step-group/README.md:16]:
```suggestion
SubStep has the same execution behavior as a normal step.
```

File: docs/examples/workflow/step-group/README.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+## Running result
```
Comment (FogDong) [inline docs/examples/workflow/step-group/README.md:18]: I think we can delete this part and the `example-result.yaml` file.

**Review by FogDong — APPROVED** [review]

**Pull request merged**